src/scss/layout/_footer.scss

```diff
@@ -0,0 +1,43 @@
+.footer {
+  background-color: $green-dark;
+
+  &__container {
+    color: $white;
+    display: flex;
+    flex-direction: column;
+    padding: 50px 34px 48px 32px;
+  }
+
+  &__title {
+    color: $white;
+  }
+
+  &__container-text {
+    width: 295px;
```

Comment:
Please organize css in alphabetical order. I believe the project has a linter to alert these issues

Reply:
Done